bash_103_combining_commands/bash_103_combining_commands.md

```diff
@@ -59,11 +59,17 @@ You will get the most out of this lesson if you follow along with the examples a
 **Open a bash shell.**
 If you are using a computer with running iOS (i.e. a Mac) you can use the **Terminal** program. If you are on a computer using Windows, open either **WLS** (Windows Linux Subsytem) or **Git Bash**. If you don't have these programs there are instructions for how to download and set them up in the [Bash / Command Line 101](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/bash_command_line_101/bash_command_line_101.md) module.
 
-<div class = "important">
-We want to be able to search, move, and rename files during this module, but don't want to do that with your important files. Therefore we will set up a little directory with a few files to experiment with. You can safely delete the whole thing afterwards if you want.
+<div class = "care">
+<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>
+
+It can be stressful to interact with files directly from the command line. Throughout this module we will make sure you know what each command does before asking you to do it, and only ask you to make changes within a practice directory that you will download.
```

Comment:
not to be the pickiest person in the world, but this still seems like it will be repetitive to read the second part of this, and then read the info outside of the box about downloading a practice directory. 

Is there a way to totally divorce the two and not mention the downloadable directory at all w/in the care box? e.g., 

```suggestion
It can be stressful to interact with files directly from the command line. Throughout this module we will make sure you know what each command does before asking you to do it.  Like any other computational skill, you'll get more comfortable with it the more you practice!
```

Feel free to totally ignore this, though -- it's just a thought I had, but I think it's also probably fine to just leave it as it currently is! :)